### DISCUSSION TITLE
Leitura de artigos em Inglês

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#643] [Discussão]
Hi guys!!

Eu sempre me virei relativamente bem na escrita em Inglês, porém quando vou ler algum artigo eu travo e não consigo ler o artigo por completo e isso vem me atrapalhando um pouco. Gostaria de saber se alguém já passou por isso? E se sim, como resolveu?

**Comment by lucianobragaweb** [comment]
Opa, tudo bom @devigor !!
Cara eu estou passando por isso. Cara tenho estudado bastante inglês, escrevo razoavelmente bem, inclusive escrevi um artigo simplesinho no Medium esses dias pra treinar.

Se quiser pode dar uma avaliada no artigo: https://medium.com/lucianobraga/adad1a4f21d6

Eu tenho buscado resolver isso, como sou autodidata, treinando bastante no DuoLingo e isso tem me ajudado muito na questão da leitura e também da interpretação da fala. Meu obstáculo principalagora é realmente falar Inglês, que é algo que ainda vai demorar um pouco.

**Comment by raulfdm** [comment]
Fala caras, tudo certo?

Então, eu sentia um pouco essa dificuldade, até vir fazer meu intercâmbio. 

Bom, a questão do intercâmbio só serviu pra me fazer estudar ingles todos os dias, mas isso não é impeditivo pra ninguém que esteja no Brasil. Enfim, vamos ao que interessa.

Eu comecei a notar que a minha compreensão de textos ficou MUITO melhor a do momento que eu comecei a ler mais em inglês.

Parece um pouco estranho, mas pensa comigo: Quanto mais você lê, mais palavras você conhece e acaba assimilando o que elas em cada situação quer dizer.

Entretanto, eu não só lia, eu sempre anotava as palavras que eu não conhecia pra depois utilizar o método de flash card.

## Método de Flashcard
Pra quem não conhece, esse método é uma forma de revisar o que você estudou em cartões. Eu conheci ele quando um amigo meu voltou dos Estados Unidos e comentou que o pessoal da universidade dele utilizava muito, para estudar todos os tipos de matérias. E isso é tão cultural, que até as crianças usam para aprender contas, palavras novas, etc.

### O cartão
Bom, imagina um cartão de papelão. Ele tem frente e verso. Na Frente dele vem uma pergunta da forma mais sucinta e resumida possível. Já no verso, vem a resposta dessa pergunta.

### Baralhos e organização
Existem várias formas de organização. A padrão é separadas primeiramente por tópicos (html, css, javascript, inglês). Depois disso, dentro de cada tema vai ter uma divisão:
- Fácil
- Médio
- Dificil
- Não feitos

### Dinâmica
Depois de organizado, quando você parar para estudar, vai pegar um cartão e tentar responder em um determinado tempo. Eu gosto de utilizar 30 segundos de limite pra responder uma pergunta. Caso passe, ele vai pro fim do baralho.

Assim, basicamente você:
1. Pega um cartão lê a pergunta;
2. Responde mentalmente;
3. Vira o cartão e confere as respostas;
4. Guarda o cartão na "Caixa" de sua dificuldade (fácil, médio, difícil)

### Repetição
Cada caixa de dificuldade tem um tempo de repetição:
- Fácil: a cada 3 a 5 dias
- Médio: 2 a 3 dias
- Difícil: 1 a 2 dias

Assim, é necessário fazer um controle do tempo que você precisa repetir os decks.

### Muito dificl isso, tem coisa mais fácil não?
Como eu disse, essa é uma metolodogia bem utilizada em muitas partes do mundo, logo, o pessoal já criou meios de estudar dessa forma, mas online e foi aí que surgiu o [Anki](https://apps.ankiweb.net/)

## ANKI
Lembra desse processo que eu descrevi acima? Ele é muito chato de se fazer na mão. Temos que cortar papeis, ter caixas, anotar quando devemos repetir cada caixa de dificuldade e blá blá. Então porque não usar um aplicativo pra isso?

O Anki originalmente era apenas um programa Desktop que se encarregava desse gerenciamento todo pra você. Porém, ele foi tão adotado pela comunidade que a mesma criou a versão web (não tão útil) e a versão mobile.

Não vou me alongar em explicar como usar e instalar o Anki porque já existe um tutorial que eu gosto muito do Mario Vergara explicando isso. Você pode conferir [aqui](http://www.mairovergara.com/como-instalar-e-usar-o-anki/)

## Exemplo Prático
Agora vou dar um exemplo de como eu vinha estudando utilizando a técnica de Flash Cards com o Anki.

Dado o texto:
```
Shu takes readers — who are presumed to have some level of proficiency with JavaScript fundamentals and jQuery — on a journey through the world of React as they build a clone of Twitter’s “compose tweet” component.
```

Eu seleciono as palavras que eu não conheço:
```
presumed
```

Entro no site no site de [Oxford](https://en.oxforddictionaries.com/), busco por ela e tento entender o significado:
```
presume
VERB

1. with clause Suppose that something is the case on the basis of probability.
````

Caso eu não entenda, entro no site [Linguee](linguee.pt/portugues-ingles) e vejo a tradução do inglês para o portugues:
```
presumed  adjetivo / particípio passado
presumido adj
presumida adj
menos frequentes:
presumidos adj ·  suspeita adj  ·  suposto adj  ·  presumíveis adj ·  considerado adj  ·  provável adj  ·  alegada adj ·  considerada adj  ·  consideradas adj ·  concluído adj  ·  estimado adj  ·  hipotético adj 
```

Então, eu crio um card mais ou menos assim:

```
Frente: 
'who are **presumed** to have some level of proficiency with JavaScript fundamentals and jQuery'

Presumed
----
Verso:
Presumido, Provável, Suposto.
```

Perceba que tem de haver um jogo de: "O que essa palavra quis dizer nesse contexto?"

## Minhas impressões sobre
Eu posso alegar com base na minha experiencia que realmente funciona. Eu conheci aprender minha palavra e até mesmo expressões usando esse método, porque a repetição das cartas quem te da é o Anki, ou seja, seu papel é dedicar todo dia algum tempo pra abrir o aplicativo e fazer.

## Conclusão
Assim como no português, no inglês existe muita repetição de palavras. A medida que você vai conhecendo e estudando as palavras e aumentando seu vocabulário, vai ver que fica cada vez mais fácil ler.

Tem muita coisa pra falar ainda, mas acredito que isso já dê um help!

Abraços

**Comment by claytonaalves** [comment]
@devigor acho que a persistência é a chave aqui.

No começo também senti muita dificuldade, mas a medida que você se liberta da necessidade de consultar um dicionário constantemente, a leitura se torna mais agradável e natural. Hoje consigo ler livros inteiros em inglês. É claro que sempre surgem palavras ou expressões novas que te forçam a consultar um dicionário.

As sugestões do @raulfdm também são muito boas. O sistema de **repetição espaçada** é muito bom para melhorar seu vocabulário.

**Comment by tcelestino** [comment]
No meu caso é ao contrário. Consigo ler e falar num boa, agora, escrever... Bom, o método que o @raulfdm deu parece ser bastante interessante, vou tentar aplicar.

Uma coisa que passei a fazer mais para continuar aumentando o vocabulário é ler bastante, acompanhar a galera no Twitter e por incrível que pareça o Reddit vem aumentando bastante meu vocabulário, principalmente por gírias. 

Um site que uso bastante é o [Linguee](http://www.linguee.com.br/) além de fazer inglês com professora particular há um ano.

O lance é não desistir e meter "as caras". :)

**Comment by carolesteves** [comment]
@devigor A sugestão do @claytonaalves é muito válida, tente não se prender àquela palavra específica que você não sabe o significado e siga com a leitura. Você vai ver que na maioria dos casos ela não vai fazer tanta diferença para o entendimento do contexto. Se ao final da frase você não tiver entendido nada do que foi lido, vale voltar e aí sim, procurar no dicionário. Com isso a leitura flui mais, e você não vai sentir como se não estivesse progredindo.

**Comment by includebrasil** [comment]
Eu utilizei o Anki no curso de Inglês do Mairo Vergara. Inclusive qm tiver interesse no curso a gente pode negociar. Pra qm ñ conhece esse é o site dele onde possui muito material interessante: http://www.mairovergara.com
